portfolio/src/main/webapp/script.js

```diff
@@ -12,29 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//Code corresponding to the enter key in a key event
```

Comment:
tiny nit: Comments should be "// start of comment" rather than "//start of comment". 

See https://google.github.io/styleguide/jsguide.html for examples.